PySDM/backends/impl_numba/methods/collisions_methods.py

```diff
@@ -89,7 +89,7 @@ def breakup_fun0(
 
 
 @numba.njit(**{**conf.JIT_FLAGS, **{"parallel": False}})
-def breakup_fun1(
+def breakup1_update_mult_attributes(
```

Comment:
What do you think about splitting this into 2 functions? Smth like `update_kth_droplet_attributes` and `get_new_multiplicities`

Reply:
we're updating j-th droplet attributes here, not k-th droplet attributes, just a matter of function naming. Would the aim of separating it be to have some common code with other attribute-modification functions within this file?

Reply:
I don't think we can easily separate the functionalities, since attributes are updated simultaneously with the multiplicities in the case that `take_from_j == multiplicity[j]`